torchreid/utils/torchtools.py

```diff
@@ -307,6 +318,20 @@ def load_pretrained_weights(model, file_path='', pretrained_dict=None):
             'please check the key names manually '
             '(** ignored and continue **)'.format(message)
         )
+
+        if len(discarded_layers) > 0:
```

Comment:
I can't understand why we need to know here about discarded layers and matched layers below if we don't have matched layers at all

Reply:
It will make much more convenient comparison between the model the state dict and the state dict loaded from a checkpoint.

E.g. before I added `_remove_prefix(k, 'nncf_module')` the weights from nncf checkpoint did not load to a model before it is wrapped by NNCF, whereas it is required as the initial stage of NNCF wrapping. 
The prints just showed what happened (`nncf_module` prefix)
Without the prints a user will have to do the same manually.